test/framework/functional/output_elasticsearch7.go

```diff
@@ -22,6 +24,7 @@ func (f *CollectorFunctionalFramework) addES7Output(b *runtime.PodBuilder, outpu
 	b.AddContainer(logging.OutputTypeElasticsearch, ElasticSearchImage).
 		AddEnvVar("discovery.type", "single-node").
 		AddRunAsUser(2000).
+		WithPullPolicy(corev1.PullIfNotPresent). // Docker image, excessive pulling will exceed quota and fail.
```

Comment:
any reason this shouldn't be always?

Reply:
Slows the build, but more importantly if you run the tests intensively as I have been doing, you hit docker's rate limiting and get flakes. Everywhere else I've switched to quay.io images, is there a quay equivalent of this one?

Reply:
My ask here is regarding the pull policy which is a kubernetes thing.  I prefer "always" for the dev experience so you make sure your changes are actually being tested.  Maybe since this is an image upon which we rely and don't devel we are OK here